test/src/test/java/org/corfudb/integration/Harness.java

```diff
@@ -46,7 +93,7 @@ private Layout getLayoutForNodes(int n) {
         long epoch = 0;
 
         for (int x = 0; x < n; x++) {
-            int port = basePort + x;
+            int port = corfuPort + x;
```

Comment:
Would be great if you rename `n` variable to something intuitive

Reply:
Done